My project/Assets/Scripts/CreatureLogic/Actions/Wander.cs

```diff
@@ -35,24 +40,24 @@ public bool EndCondition()
     public void OnEnter()
     {
         wander_target = data.SetRandomPath();
-        while(wander_target == Vector3Int.zero){
-            wander_target = data.SetRandomPath();
-        }
-        creature_movement_reworked
+        forceQuit = false;
         Vector3Int grid_position = GameManager.Instance.getGrid().WorldToCell(rb.position);
-        rb.velocity = new Vector2(wander_target.x - grid_position.x, wander_target.y - grid_position.y).normalized * data.Speed * .02f;
-
+        rb.velocity = new Vector2(wander_target.x - grid_position.x, wander_target.y - grid_position.y).normalized * data.Speed;
+        Debug.Log(rb.velocity);
```

Comment:
camel case? i know there might be some weird rebase merge conflict with your other PR